asteroid/filterbanks/__init__.py

```diff
@@ -80,7 +81,8 @@ def get(identifier):
 analytic_free = AnalyticFreeFB
 param_sinc = ParamSincFB
 stft = STFTFB
+multiphase_gammatone_filterbank = MultiphaseGammatoneFB
```

Comment:
We removed the `filterbank` on the other names, maybe you can have something like 
`multiphase_gammatone = MultiphaseGammatoneFB`
or even `multiphase_gammatone = mpgtf = MultiphaseGammatoneFB` so that you have long and short alias.